openreview/openreview.py

```diff
@@ -419,26 +428,26 @@ def post(self, client):
         client.post_group(self)
 
 class Invitation(object):
-    def __init__(self, id, writers=None, invitees=None, noninvitees=None, readers=None, nonreaders=None, reply=None, web=None, process=None, signatures=None, duedate=None, cdate=None, rdate=None, ddate=None, multiReply=None, taskCompletionCount=None):
+    def __init__(self, id, writers=None, invitees=None, noninvitees=None, readers=None, nonreaders=None, reply=None, replyto=None, forum=None, web=None, process=None, signatures=None, duedate=None, cdate=None, rdate=None, ddate=None, multiReply=None, taskCompletionCount=None):
 
         default_reply = {
-            'forum': None,
-            'replyto': None,
+            'forum': forum,
+            'replyto': replyto,
             'readers': {'values': ['everyone']},
-            'signatures': {'values-regex': '.*'},
-            'writers': {'values-regex': '.*'},
+            'signatures': {'values-regex': '~.*'},
```

Comment:
this might require a longer conversation... I can take out the defaults for now, though.

The reason I wanted to add defaults is because I think there are certain group parameters that are much more common than others, and it can reduce the amount of boilerplate code. but yes, we have to do this very carefully